client/css/style.css

```diff
@@ -681,19 +665,38 @@ button {
 #chat.hide-quit .quit {
 	display: none !important;
 }
-#chat .notice .type {
-	display: none;
+#chat .join .from:before {
+	content: "→";
+	color: #2ECC40;
 }
-#chat .action,
+#chat .kick .from:before,
+#chat .quit .from:before,
+#chat .part .from:before {
+	content: "←";
+	color: #FF4136;
+}
+#chat .topic .from:before,
+#chat .mode .from:before,
+#chat .nick .from:before {
+	content: "•";
+	color: #2ECC40;
+}
+#chat .action .text,
 #chat .chan .action .user {
 	color: #f39c12 !important;
 }
-#chat .action .user:before {
-	content: "* ";
+#chat .action .from:before {
+	content: "★";
```

Comment:
Just a random thought, what about using Font Awesome (as we do everywhere else) for these icon-like goodies?